### ISSUE TITLE
Page objects created via PDF Issue upload missing date_uploaded fields

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#74]
**As an admin user**
When I create a new NewspaperIssue work via the admin UI
And I upload a PDF as an attachment to that work
The NewspaperPage objects that are created representing the pages of the issue
Should have a "Date Added" value in the Hyrax 'Works' dashboard

**Discussion**
The 'Works' view in Hyrax admin dashboard UI has a "Date Added" column. This field is populated for the Issue work, but not the child Page works.

Looking at the Issue and Page objects in the console, it seems that Page objects have `nil` values for `date_uploaded` and `date_modified` properties:
```
irb(main):001:0> iss = NewspaperIssue.find("db78tc01m")
=> #<NewspaperIssue id: "db78tc01m", head: [#<ActiveTriples::Resource:0x32ad610 ID:<http://127.0.0.1:8984/rest/dev/db/78/tc/01/db78tc01m/list_source#g70117091138600>>], tail: [#<ActiveTriples::Resource:0x31ec99c ID:<http://127.0.0.1:8984/rest/dev/db/78/tc/01/db78tc01m/list_source#g70117089081920>>], depositor: "testy@test.com", title: ["Issue for IIIF Testing"], date_uploaded: "2018-11-16 19:33:58", date_modified: "2018-11-16 19:33:58", state: #<ActiveTriples::Resource:0x31ac540 ID:<http://fedora.info/definitions/1/0/access/ObjState#active>>, proxy_depositor: nil, on_behalf_of: nil, arkivo_checksum: nil, owner: nil, alternative_title: [], genre: ["text"], place_of_publication: [], issn: nil, lccn: nil, oclcnum: nil, held_by: "Springvale Public Library", volume: nil, edition: nil, issue_number: nil, extent: nil, publication_date: nil, label: nil, relative_path: nil, import_url: nil, resource_type: ["Journal"], creator: [], contributor: [], description: [], keyword: [], license: [], rights_statement: [], publisher: [], date_created: [], subject: [], language: ["English"], identifier: [], based_near: [], related_url: [], bibliographic_citation: [], source: [], access_control_id: "fc580b60-81c5-420e-aa0e-26fa648f4a18", representative_id: "q811kj606", thumbnail_id: "q811kj606", rendering_ids: [], admin_set_id: "admin_set/default", embargo_id: nil, lease_id: nil>

irb(main):002:0> page = NewspaperPage.find("f1881k888")
=> #<NewspaperPage id: "f1881k888", head: [#<ActiveTriples::Resource:0x4178960 ID:<http://127.0.0.1:8984/rest/dev/f1/88/1k/88/f1881k888/list_source#g70116789112760>>], tail: [#<ActiveTriples::Resource:0x400f718 ID:<http://127.0.0.1:8984/rest/dev/f1/88/1k/88/f1881k888/list_source#g70116789112760>>], depositor: "testy@test.com", title: ["Page 2"], date_uploaded: nil, date_modified: nil, state: nil, proxy_depositor: nil, on_behalf_of: nil, arkivo_checksum: nil, owner: nil, text_direction: nil, page_number: nil, section: nil, height: nil, width: nil, label: nil, relative_path: nil, import_url: nil, resource_type: [], creator: [], contributor: [], description: [], keyword: [], license: [], rights_statement: [], publisher: [], date_created: [], subject: [], language: [], identifier: [], based_near: [], related_url: [], bibliographic_citation: [], source: [], access_control_id: "3ea4fe8b-5d6a-465d-a9d9-6fe4d867c6a6", representative_id: "sf268508b", thumbnail_id: "sf268508b", rendering_ids: [], admin_set_id: "admin_set/default", embargo_id: nil, lease_id: nil>

```
We need to investigate how those values are set.

**This ticket will be complete when:**
Automatically-created Page objects have correct values for `date_uploaded` and `date_modified` properties.

**Comment by seanupton** [comment]
@ebenenglish does testing for non-nil date values on the created child pages seem sufficient?  See: https://github.com/marriott-library/newspaper_works/blob/04462045f90ceaae6d00f743193caa6489b85890/spec/lib/newspaper_works/ingest/newspaper_issue_ingest_spec.rb#L35-L38

**Comment by ebenenglish** [comment]
@seanupton That seems fine.

**Comment by ebenenglish** [comment]
Fixed via #127.
